### PR TITLE
Eslint error fix

### LINKED ISSUE #103: Bring ESLint and testing suite to pass
We currently have a failing testing suite because of the addition of ESLint to our codebase. Please fix all ESLint related issues and do your best to bring our codebase up to speed with tests.

### PR DESCRIPTION
**Pull request by rav7kantsingh**
#103 
@cereallarceny 
@vvmnnnkv

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
     }
   },
   "dependencies": {
+    "babel-eslint": "^10.1.0",
```
Comment (vvmnnnkv) [inline package.json]: Move to devdependencies

File: test/data_channel_message.test.js [hunk 2]
```diff
@@ -42,17 +43,17 @@ describe('Data Channel Message', () => {
...
+  test('can assemble full message from chunks', done => {
     const buf = new ArrayBuffer(WEBRTC_DATACHANNEL_CHUNK_SIZE + 100);
     randomFillSync(new Uint8Array(buf), 0, buf.byteLength);
-    const messageOrig = new DataChannelMessage({data: buf});
+    const messageOrig = new DataChannelMessage({ data: buf });
     const chunk1 = messageOrig.getChunk(0);
     const chunk2 = messageOrig.getChunk(1);
     const info1 = DataChannelMessage.messageInfoFromBuf(chunk1);
-    const info2 = DataChannelMessage.messageInfoFromBuf(chunk2);
+    // const info2 = DataChannelMessage.messageInfoFromBuf(chunk2);
```
Comment (vvmnnnkv) [inline test/data_channel_message.test.js]: Can be just removed if not used anywhere

File: test/data_channel_message.test.js [hunk 3]
```diff
@@ -67,21 +68,20 @@ describe('Data Channel Message', () => {
   test('should error on invalid chunks', () => {
     const buf = new ArrayBuffer(WEBRTC_DATACHANNEL_CHUNK_SIZE + 100);
     randomFillSync(new Uint8Array(buf), 0, buf.byteLength);
-    const messageOrig = new DataChannelMessage({data: buf});
+    const messageOrig = new DataChannelMessage({ data: buf });
     const chunk1 = messageOrig.getChunk(0);
-    const chunk2 = messageOrig.getChunk(1);
+    // const chunk2 = messageOrig.getChunk(1);
```
Comment (vvmnnnkv) [inline test/data_channel_message.test.js]: Can be just removed if not used anywhere

File: test/data_channel_message_queue.test.js [hunk 4]
```diff
@@ -30,27 +28,27 @@ describe('Data Channel Message Queue', () => {
...
+  test('emits "message" event when the message is ready', done => {
     const buf = new ArrayBuffer(WEBRTC_DATACHANNEL_CHUNK_SIZE + 100);
     randomFillSync(new Uint8Array(buf), 0, buf.byteLength);
-    const messageOrig = new DataChannelMessage({data: buf});
+    const messageOrig = new DataChannelMessage({ data: buf });
     const chunk1 = messageOrig.getChunk(0);
     const chunk2 = messageOrig.getChunk(1);
     const info1 = DataChannelMessage.messageInfoFromBuf(chunk1);
-    const info2 = DataChannelMessage.messageInfoFromBuf(chunk2);
+    // const info2 = DataChannelMessage.messageInfoFromBuf(chunk2);
```
Comment (vvmnnnkv) [inline test/data_channel_message_queue.test.js]: Can be just removed

**Review by vvmnnnkv — CHANGES_REQUESTED** [review]

**Comment by cereallarceny** [comment]
For the record, we won't be merging this PR until we have 100% passing tests. We don't need to have 100% test coverage, but we need all failing tests and linting errors to be resolved.

**Comment by rav7kantsingh** [comment]
> For the record, we won't be merging this PR until we have 100% passing tests. We don't need to have 100% test coverage, but we need all failing tests and linting errors to be resolved.

This pr only contains eslint error fix and for jest test fails I am working on another Pr #105

**Review by vvmnnnkv — APPROVED** [review]

**Comment by vvmnnnkv** [comment]
Thanks!

**Pull request merged**